dataactvalidator/scripts/load_cfda.py

```diff
@@ -1,54 +1,89 @@
-from ftplib import FTP
+import os
+from datetime import datetime, timedelta
 from io import BytesIO
 from dataactcore.config import CONFIG_BROKER
 from dataactcore.logging import configure_logging
+import logging
 
 import boto3
+from botocore.handlers import disable_signing
+from botocore.exceptions import ClientError
+
+s3 = boto3.client('s3', )
+logger = logging.getLogger(__name__)
+
+CFDA_FILE_FORMAT = os.path.join('Assistance Listings', 'usaspendinggov', '%Y', '%m-%b',
+                                'AssistanceListings_USASpendingGov_PUBLIC_WEEKLY_%Y%m%d.csv')
+WEEKDAY_UPLOADED = 5  # datetime.weekday()'s integer representing the day it's usually uploaded (Saturday)
+DAYS_TO_SEARCH = 4 * 7  # 4 weeks
+LOCAL_CFDA_FILE = os.path.join('dataactvalidator', 'config', 'cfda_program.csv')
+LOCAL_CFDA_FILE2 = os.path.join('dataactvalidator', 'config', 'cfda_program2.csv')
```

Comment:
This doesn't seem to be used anywhere in the script, is it necessary to have here?

Reply:
Whoops, that's an artifact from testing. Removing.